silabs-multiprotocol/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.4.6
+- Formally deprecate the addon. It will continue running as usual but you should find an alternative way to run both Zigbee and Thread if you use both with the same adapter.
```

Comment:
The changelog will be shown quite prominently for most users on upgrade. Ideally we use this to link to instructions how users can move their installation to Zigbee/Thread only.